daffodil-test/src/test/resources/org/apache/daffodil/section12/delimiter_properties/DelimiterProperties.tdml

```diff
@@ -546,7 +546,7 @@
   </tdml:defineSchema>
 
   <tdml:parserTestCase name="initiatedContentSimple1"
-    model="initiatedContentSimple1" description="TBD" root="e">
+    model="initiatedContentSimple1" description="TBD" root="e" roundTrip="false">
```

Comment:
Why doesn't this round trip now?

Reply:
That is a good question.  I think I had set it to roundTrip="false" while debugging some things and I thought it would work now, but now it fails with an SDE saying No root element found, which is not what I would have expected.

Reply:
Nevermind, I simply accidentally deleted the root="e" when I was removing the roundTrip attribute.  This test works and roundTrips as expected.